R/expected_time.R

```diff
@@ -120,3 +112,24 @@ expected_time <- function(
     return(ans)
   }
 }
+
+#' Find the difference between `ahr()` and different values of `total_duration`
```

Comment:
I personally suggest rewording it as follows: 'Considering the enrollment rate, failure rate, and randomization ratio, calculate the difference between the targeted number of events and the accumulated events at time `x`.' Please review and let me know if you agree with this rephrased version.

Reply:
I agree that's a great title! I just pushed the change